tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml

```diff
@@ -1,79 +1,58 @@
 kube-prometheus-stack:
   prometheus:
     prometheusSpec:
-      scrapeInterval: 30s
```

Comment:
Why this file contains changes not related to remoteTimeout?

Reply:
Good catch! Should be fixed now.

Reply:
It seems that now changes are related only to remoteTimeout but it different order. May we avoid reordering? I'll be easier to review.

Reply:
I guess the reason for this is how `yq` approaches yaml maps/hashmaps/? 🤷